### PR TITLE
 Add support for missing metadata types

### PR DESCRIPTION
**Pull request by jayree**
+ add support for: flows, reports, dashboards, translations, ...
+ parameter to exclude managed packages from the output
+ update dependencies

### CODE REVIEW

**Comment by mikesimps** [comment]
This is great @jayree ! I could not figure out how to exclude managed packages, so that is a great addition. What did you do to add support for flows/dashboard/etc? Based on my understanding those already worked.

**Comment by jayree** [comment]
Thanks @mikesimps.  I can't remember exactly what was wrong with reports and dashboards. I thought it was related to the call with "folder: null". But it doesn't change anything if you revert it back. Perhaps it was resolved by updating the dependencies or updating the api version. Need to review that later.

For standardValueSetTranslations and globalValueSetTranslations the returned type for the object is "{ '$': { 'xsi:nil': 'true' } }" so I used the filename extension to set the correct type.

For Flows the returned name for an activated flow is "flow" instead of "flow-1" if version "1" is activated. As the (file)name "flow" doesn't exist, I've fixed that using the Tooling API retrieving the active version for all flows (FlowActiveVersionPromise) and set the correct name.

**Comment by mikesimps** [comment]
Thanks @jayree . I've tried your version and I'm not seeing where it is excluding InstalledPackages from the metadata. Have you tried this on multiple types of orgs?

**Comment by jayree** [comment]
@mikesimps have you used -x as parameter? This parameter excludes currently only "Managed Packages" (metadataEntries.manageableState = installed) "Unmanaged Packages" are still included (metadataEntries.manageableState = unmanaged)

As I found out this morning looking into the issue #14 of this project, it excludes currently only the components of the package not the package itself in the InstalledPackage type. I need to work on that.

**Comment by mikesimps** [comment]
Using the Non-profit Service Pack as my example, I unfortunately still saw items in CustomObjectTranslation and TopicsForObjects types. Not sure what is special about them, but if you could look at that as well that would be great!

If you can get that working, I can do the work to make it compatible with the config file and update the documentation.

File: README.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ This will produce an XML string, and pipe it directly into the file called packa
 |---|---|
 |-a, --api|Set the API version of the packagexml file (Default is 42.0)|
```
Comment (mikesimps) [inline README.md]: @jayree can you update this line to say the default is now 43?

Comment (jayree) [reply README.md]: done

**Comment by jayree** [comment]
@mikesimps I've checked the API again regarding the items in CustomObjectTranslation and TopicsForObjects types. For me it seems that these items were created after the package was installed (either manual or by the package itself). I'm not sure if we should remove these items as they are a kind of "content" of the package and not "metadata" wich will be installed with the package. What do you think?

**Comment by mikesimps** [comment]
@jayree Based on my understanding, COT and TFO are both similar to Custom Metadata types which is basically what you described - content stored in metadata. For those who use these features would not want them excluded so I recommend leaving them in.

**Pull request merged**